devito/operator.py

```diff
@@ -359,17 +364,21 @@ def _profile_output(self, args):
         summary = self.profiler.summary(args, self._dtype)
         with bar():
             for k, v in summary.items():
-                name = '%s<%s>' % (k, ','.join('%d' % i for i in v.itershape))
-                gpointss = ", %.2f GPts/s" % v.gpointss if k == 'main' else ''
-                info("Section %s with OI=%.2f computed in %.3f s [%.2f GFlops/s%s]" %
+                itershapes = [",".join(str(i) for i in its) for its in v.itershapes]
+                if len(itershapes) > 1:
+                    name = "%s<%s>" % (k, ",".join("<%s>" % i for i in itershapes))
```

Comment:
I love that it lists all the itershapes now.  That makes it much clearer what's included in each one (in the case of BCs).